### PR TITLE
Adding 'MinApp' test apps for perf comparisons

### PR DESCRIPTION
**Pull request by jaredhms**

### CODE REVIEW

File: tests/testapps/MinAppCppCx/MinAppCppCx/MinAppCppCx.vcxproj [hunk 1]
```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bbowman) [inline tests/testapps/MinAppCppCx/MinAppCppCx/MinAppCppCx.vcxproj]: You'll want to use PackageReference style nuget deps with the override targets here if you want to use the latest / local packages.

File: tests/testapps/MinApp/MinApp.vsimporter/MinApp-WinStore10/project.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "WinObjC.Frameworks": "*",
```
Comment (bbowman) [inline tests/testapps/MinApp/MinApp.vsimporter/MinApp-WinStore10/project.json]: You'll want to move to PackageReference instead. Additionally, you'll want to add all packages (except tools and packaging) here to handle cases where you update a single pacakge that isn't a leaf node and want to test local updates.

Comment (bbowman) [inline tests/testapps/MinApp/MinApp.vsimporter/MinApp-WinStore10/project.json]: Look at WOCCatalog as an example.

Comment (jaredhms) [inline tests/testapps/MinApp/MinApp.vsimporter/MinApp-WinStore10/project.json]: Why didn't vsimporter 'do the right thing' here?

Comment (bbowman) [inline tests/testapps/MinApp/MinApp.vsimporter/MinApp-WinStore10/project.json]: @jaredhms it did for a normal app developer. This is not a normal app. The two leaf nodes with a "*" version is exactly what we want most external developers to use as it is the simplest way to have them stay up to date on the latest pre-release version.

We don't currently have a switch in VSImporter to do version swapping but it is something we know about and are working on. See https://github.com/Microsoft/WinObjC/issues/2199

Comment (jaredhms) [inline tests/testapps/MinApp/MinApp.vsimporter/MinApp-WinStore10/project.json]: ah, that's right - thanks for the clarification.

---
In reply to: [107011503](https://github.com/Microsoft/WinObjC/pull/2278#discussion_r107011503) [](ancestors = 107011503)

**Comment by yiyang-msft** [comment]
:shipit:

**Pull request merged**